### PR TITLE
[FrameworkBundle] Make TemplateController working without the Templating component

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

The template controller was missed during #15502. Btw I've added some tests for this controller.

### CODE REVIEW

**Review by Simperfit — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @dunglas.

File: src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php [hunk 1]
```diff
@@ -33,8 +33,14 @@ class TemplateController extends ContainerAware
      */
     public function templateAction($template, $maxAge = null, $sharedAge = null, $private = null)
     {
-        /** @var $response \Symfony\Component\HttpFoundation\Response */
-        $response = $this->container->get('templating')->renderResponse($template);
+        /* @var $response \Symfony\Component\HttpFoundation\Response */
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php]: @dunglas Are you sure about this change?

Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php]: I would even remove it as I don't think it helps in any way, does it?

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php]: Removing it would indeed be better.

Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php]: done in e5ce4c9

Comment (dunglas) [inline src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php]: It was to make fabbot green, I've tried with PHPStorm, both formats enable the autocompletion. Removing it prevent autocompletion by default, but if you have the Symfony plugin then it works.

**Pull request merged**